CMakeLists.txt

```diff
@@ -193,6 +193,9 @@ set (Boost_MULTITHREADED TRUE)
 
 if (USE_STATIC_BOOST)
     set (Boost_USE_STATIC_LIBS TRUE)
+    add_definitions(-DBOOST_PYTHON_STATIC_LIB) # link statically against Boost.Python
```

Comment:
That used to be defined on Windows only, now it's going to be defined on all platforms. Not sure what the impact is. @est77?

Please add a space after `add_definitions`.

Reply:
BOOST_PYTHON_STATIC_LIB doesn't seem to be used on unix, at least in boost 1.61.

Reply:
Ok, thanks. Might still be cleaner to restrict this to MSVC...